script.js

```diff
@@ -18,4 +18,233 @@ mobileLinks.forEach((link) => {
+
+                          <div class="popup-icons">
+                            <img src="./Images/Icon -GitHub.png" alt="see-live-icon">
+                          </div>
+                        </button>
+                      </div>
+                    <div class="project-popup-title">
+                    <h3>${cardData[0].popTitle}</h3>
+                    <ul class="popup-project-abilites2">
+                          <li class="a1">${cardData[0].popTechnologies2[0]}</li>
+                          <li class="a1">${cardData[0].popTechnologies2[1]}</li>
+                          <li class="a1">${cardData[0].popTechnologies2[2]}</li>
+                          <li class="a1">${cardData[0].popTechnologies2[3]}</li>
+                          <li class="a1">${cardData[0].popTechnologies2[4]}</li>
+                          <li class="a1">${cardData[0].popTechnologies2[5]}</li>
+                      </ul>
+
+                    <ul class="popup-project-abilites">
+                        <li class="a1">${cardData[0].popTechnologies1[0]}</li>
+                        <li class="a1">${cardData[0].popTechnologies1[1]}</li>
+                        <li class="a1">${cardData[0].popTechnologies1[2]}</li>
+                    </ul> 
+                    </div>
+                    
+                    <p class="mobile-popup-text">
+                        ${cardData[0].capstoneIntro1}
+                    </p>
+                    <div class="mobile-popup-buttons">
+                    <a href=${cardData[0].linkToLiveSource}>
+                    <button class="btn">
+                        <div class="popup-btn-text">${cardData[0].btn1}</div>
+                        <div class="popup-icons">
+                        <img src="./Images/see live icon.png" alt="see-live-icon">
+                        </div>
+                    </button>
+                    </a>
+                    <a href=${cardData[0].linkToSource}>
+                    <button class="btn">
+                        <div class="popup-btn-text">${cardData[0].btn2}</div>
+                        <div class="popup-icons">
+                        <img src="./Images/Icon -GitHub.png" alt="see-live-icon">
+                        </div>
+                    </button>
+                    </a>
+                    </div>
+                </div>`;
+
+  popWindow1.innerHTML = popScreen;
+}
+
+popDisplay();
+// popup functions goes there
+const openMobilePopUp = document.querySelectorAll('[data-modal-target]');
+const closeMobilePopUp = document.querySelectorAll('[data-close-button]');
+const overlay = document.getElementById('overlay');
+
+function openModal(modal) {
+  if (modal === null) return;
+  modal.classList.add('active');
+  overlay.classList.add('active');
+}
+
+openMobilePopUp.forEach((button) => {
+  button.addEventListener('click', () => {
+    const modal = document.querySelector(button.dataset.modalTarget);
+    openModal(modal);
+  });
+});
+
```

Comment:
- [ ] Well done :clap: you did a very nice job :+1: but there are some little issues please stick to the Figma design. If you check your design popup background is not match with Figma design Kindly fix it. Check below :point_down: the comparison between your design and FIgma design :slightly_smiling_face: 

| Your Desgin   |      Figma Design      |
|---------- |:----------------:|
| ![Screenshot from 2023-04-19 00-58-38](https://user-images.githubusercontent.com/45226848/232892624-01703611-4601-47c5-88ba-514a1626ac14.png) |  ![Screenshot from 2023-04-19 00-59-08](https://user-images.githubusercontent.com/45226848/232892656-894872b3-ec99-4e4c-963f-c34caf068d95.png) |